Source/Editor/Foundation/ProfilerTab.cpp

```diff
@@ -119,7 +119,7 @@ void ProfilerTab::RenderContent()
         ui::SameLine();
         bool connect = ui::InputText("", &connectTo_, ImGuiInputTextFlags_EnterReturnsTrue);
         ui::SameLine();
-        connect |= ui::Button(ICON_FA_WIFI " Connect");
+        connect |= ui::Button((const char*)(ICON_FA_WIFI " Connect"));
```

Comment:
Whats the cast for? With or without cast resulting type should still be `const char*`.